CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj rebase` now accepts multiple `-s` and `-b` arguments. Revsets with
   multiple commits are allowed with `--allow-large-revsets`.
 
+* `jj git fetch` now supports a `--branch` argument to fetch some of the
+  branches only. For now, there are a few known bugs.
```

Comment:
Is it still true (the known bugs)?

Reply:
Thanks for catching this! There is still one outstanding bug with `undo`, but it's too minor to mention it here.